packages/mux-uploader/src/mux-uploader-progress.ts

```diff
@@ -111,35 +113,36 @@ class MuxUploaderProgressElement extends globalThis.HTMLElement {
     this.setDefaultType();
 
     this.#uploaderEl = getMuxUploaderEl(this);
+    this.#controller = new AbortController();
```

Comment:
Might be worth naming it `#abortController`. Since we also have an `uploaderController`, it would make it easier not to get confused between the two.